egui/src/data/input.rs

```diff
@@ -123,6 +123,25 @@ pub enum Event {
     CompositionUpdate(String),
     /// IME composition ended with this final result.
     CompositionEnd(String),
+
+    Touch {
+        /// Hashed device identifier (if available; may be zero).
+        /// Can be used to separate touches from different devices.
+        device_id: TouchDeviceId,
+        /// Unique identifier of a finger/pen.  Value is stable from touch down
+        /// to lift-up
+        id: TouchId,
+        phase: TouchPhase,
+        /// Position of the touch (or where the touch was last detected)
+        pos: Pos2,
+        /// Describes how hard the touch device was pressed. May always be `0` if the platform does
+        /// not support pressure sensitivity.
+        /// The value is in the range from 0.0 (no pressure) to 1.0 (maximum pressure).
+        force: f32,
+        // ### Note for review: using f32 forced me to remove `#[derive(Eq)]`
+        //     from the `Event` struct.  Can this cause issues?  I did not get errors,
+        //     so I wonder if `Eq` had been derived on purpose.
```

Comment:
I think it's fine to remove `Eq`. Can't really see any real reason to have it

Reply:
ok, thanks. I removed these code comments.